dev-test/config.yml

```diff
@@ -6,6 +6,11 @@ site_url: https://example.com
 publish_mode: editorial_workflow
 media_folder: assets/uploads
 
+i18n:
```

Comment:
Can we revert these changes and the ones to `index.html`?
This is what gets deployed to 
https://cms-demo.netlify.com/#/collections/posts
See https://deploy-preview-4487--cms-demo.netlify.app/#/collections/i18n_settings

I don't think we want it as a part of the demo site (yet)